Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1500,6 +1500,10 @@ class BrowserViewController: UIViewController {
             // forward/backward. Strange, but LayoutChanged fixes that.
             UIAccessibility.post(notification: .layoutChanged, argument: nil)
         } else if let webView = tab.webView {
+            // For some reason the web view will steal first responder as soon
+            // as its added to the view heirarchy below. This line fixes that...
+            // somehow...
```

Comment:
Can you add ticket number to this comment for reference?

Reply:
Sure